arp_scripts/submit_jt_cal.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source /reg/g/pcds/engineering_tools/xpp/scripts/pcds_conda
```

Comment:
Why xpp?

Reply:
Historical: I've defaulted to XPP as I had control over where that pointed and I did the smalldata_tools stuff. It should probably use "expname[0:3]" instead of xpp or point to latest_release (whatever that directory was called again).